apps/dotcom/src/components/StoreErrorScreen.tsx

```diff
@@ -26,6 +26,11 @@ export function StoreErrorScreen({ error }: { error: Error }) {
 					'Your changes were rejected by the server. Please reload the page. If the problem persists contact the system administrator.'
 				break
 			}
+			case TLIncompatibilityReason.RoomNotFound: {
```

Comment:
Should we maybe reuse our page `ErrorPage` component for this case? Feels more related to page not found (like [this one](https://www.tldraw.com/asdfasdf)) than to a store error?

Reply:
I wouldn't disagree - it's just that the `<ErrorPage>` isn't really wire up in the path here. 

It's really in `MultiplayerEditor` (which has a reference to StoreErrorScreen and the store is what gets the error). I think this is ok actually. One could see our client, say, having a list of bookmarks of rooms to connect to and maybe clicking on some don't work but some do. So I think the _client_ doesn't 404, it's more that the _room_ has the 404. In theory, the client could keep going if it wanted to.

Reply:
> I wouldn't disagree - it's just that the `<ErrorPage>` isn't really wired up in the path here.
>
> It's really in `MultiplayerEditor` (which has a reference to StoreErrorScreen and the store is what gets the error).

Yeah, we could handle that case in `MultiplayerEditor` 🤷‍♂️ I guess I'm still thinking of this more in the traditional rest routes `/r/id` points to a certain room, but that room does not exist so I'd expect a 404 error. Still ok with this is you wish to keep it as is.